src/pages/studyView/styles.module.scss

```diff
@@ -1,10 +1,21 @@
-.flexContainer {
+.studyViewFlexContainer {
     display: inline-flex;
     flex-direction: row;
     flex-wrap: wrap;
     width: 100%;
 }
-.survivalPlot {
+.studyViewSurvivalPlot {
     float: left;
```

Comment:
we can clean this up loater but we shouldn't need float anymore.  In a flex box layout, float has no impact.